supertokens_python/recipe/emailpassword/emaildelivery/service/smtp/service_implementation/__init__.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from typing import Any, Dict
+
+from supertokens_python.ingredients.emaildelivery.service.smtp import (
+    GetContentResult, ServiceInterface, SMTPServiceConfigFrom, Transporter)
+from supertokens_python.recipe.emailpassword.emaildelivery.service.smtp.password_reset_implementation import \
+    get_password_reset_email_content
+from supertokens_python.recipe.emailpassword.types import \
+    TypeEmailPasswordEmailDeliveryInput
+from supertokens_python.recipe.emailverification.emaildelivery.service.smtp.implementation import \
+    ServiceImplementation as EVServiceImplementation
+from supertokens_python.recipe.emailverification.interfaces import \
+    TypeEmailVerificationEmailDeliveryInput
+
+from .email_verification_implementation import \
+    ServiceImplementation as DerivedEVServiceImplementation
+
+
+class ServiceImplementation(ServiceInterface[TypeEmailPasswordEmailDeliveryInput]):
+    def __init__(self, transporter: Transporter) -> None:
+        self.transporter = transporter
+
+        email_verification_service_implementation = EVServiceImplementation(transporter)
+        self.ev_send_raw_email = email_verification_service_implementation.send_raw_email
+        self.ev_get_content = email_verification_service_implementation.get_content
+
+        derived_ev_service_implementation = DerivedEVServiceImplementation(self)
+        email_verification_service_implementation.send_raw_email = derived_ev_service_implementation.send_raw_email
+        email_verification_service_implementation.get_content = derived_ev_service_implementation.get_content
+
+    async def send_raw_email(self, get_content_result: GetContentResult, config_from: SMTPServiceConfigFrom, user_context: Dict[str, Any]) -> None:
+        self.transporter.send_email(config_from, get_content_result, user_context)
```

Comment:
Are you sure that we don't have to await this? How will this behave in an event loop driven runtime env?

Reply:
You're right. Using `await` now.